src/javascript/JContent/ContentRoute/ContentLayout/FilesGrid/FileCard/FileSize/FileSize.test.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import FileSize from './FileSize';
+import {shallow} from '@jahia/test-framework';
+
+describe('FileSize', () => {
+    it('should show the size', () => {
+        const mockNode = {
+            children: {
+                nodes: [
+                    {
+                        data: {
+                            size: 100
+                        }
+                    }
+                ]
+            }
+        };
+        const wrapper = shallow(<FileSize node={mockNode}/>);
+        expect(wrapper).toBeTruthy();
```

Comment:
Is there a scenario where that one could fail? If not, please remove that line.

Reply:
sure